src/Microsoft.IdentityModel.Tokens/Validation/Validators.TokenType.cs

```diff
@@ -54,19 +70,12 @@ internal static TokenTypeValidationResult ValidateTokenType(string? type, Securi
                         new StackFrame(true)));
             }
 
-            if (validationParameters.TypeValidator == null && (validationParameters.ValidTypes == null || !validationParameters.ValidTypes.Any()))
+            if (validationParameters.ValidTypes == null || validationParameters.ValidTypes.Count == 0)
```

Comment:
If ValidationParameters.ValidTypes cannot be null, then the null check is not needed.

Reply:
Removed, thanks @brentschmaltz!